sw/device/tests/sim_dv/alert_handler_escalation.c

```diff
@@ -50,7 +50,6 @@ typedef struct node {
   dif_alert_handler_class_t class;
 } node_t;
 
-static volatile uint32_t global_alert_called;
```

Comment:
@timothytrippel IIRC some of these `static volatile`s are backdoor-loaded by the testbench. Can you also take a look at this file?

Reply:
For the sake of readability and silencing warnings we should probably define a macro like `OT_VOLATILE_FOR_DV` in `macros.h`

Reply:
Thanks, @timothytrippel!

Reply:
The test passes. This change is ok.